[2022-05-13 15:28:14,908] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: my_dag_v0.0.1.my_task_2 2022-05-10T00:00:00+00:00 [queued]>
[2022-05-13 15:28:14,920] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: my_dag_v0.0.1.my_task_2 2022-05-10T00:00:00+00:00 [queued]>
[2022-05-13 15:28:14,920] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-05-13 15:28:14,921] {taskinstance.py:1068} INFO - Starting attempt 1 of 4
[2022-05-13 15:28:14,921] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-05-13 15:28:14,927] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): my_task_2> on 2022-05-10T00:00:00+00:00
[2022-05-13 15:28:14,931] {standard_task_runner.py:52} INFO - Started process 383 to run task
[2022-05-13 15:28:14,933] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'my_dag_v0.0.1', 'my_task_2', '2022-05-10T00:00:00+00:00', '--job-id', '29', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag_v_0_1_0_0.py', '--cfg-path', '/tmp/tmpv31szyj5', '--error-file', '/tmp/tmpo5e50ae0']
[2022-05-13 15:28:14,935] {standard_task_runner.py:77} INFO - Job 29: Subtask my_task_2
[2022-05-13 15:28:14,996] {logging_mixin.py:104} INFO - Running <TaskInstance: my_dag_v0.0.1.my_task_2 2022-05-10T00:00:00+00:00 [running]> on host bacb497a28fb
[2022-05-13 15:28:15,090] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=msbeni@gmail.com
AIRFLOW_CTX_DAG_OWNER=mohasa
AIRFLOW_CTX_DAG_ID=my_dag_v0.0.1
AIRFLOW_CTX_TASK_ID=my_task_2
AIRFLOW_CTX_EXECUTION_DATE=2022-05-10T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-10T00:00:00+00:00
[2022-05-13 15:28:15,091] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2022-05-13 15:28:15,091] {subprocess.py:63} INFO - Running command: ['bash', '-c', "echo '1' && exit 0"]
[2022-05-13 15:28:15,099] {subprocess.py:75} INFO - Output:
[2022-05-13 15:28:15,100] {subprocess.py:79} INFO - 1
[2022-05-13 15:28:15,101] {subprocess.py:83} INFO - Command exited with return code 0
[2022-05-13 15:28:15,126] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=my_dag_v0.0.1, task_id=my_task_2, execution_date=20220510T000000, start_date=20220513T152814, end_date=20220513T152815
[2022-05-13 15:28:15,215] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-05-13 15:28:15,227] {local_task_job.py:151} INFO - Task exited with return code 0
